src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -956,6 +963,32 @@ internal void ChangeMutationPoints(int change)
         MutationPoints = (MutationPoints + change).Clamp(0, Constants.BASE_MUTATION_POINTS);
     }
 
+    /// <summary>
+    ///   Recursively loops though all hexes and checks if there any without connection to the rest.
+    /// </summary>
+    /// <returns>
+    ///   Returns a list of hexes that are not connected to the rest
+    /// </returns>
+    internal List<Hex> GetIslandHexes()
+    {
+        // All organelles
+        var organelles = editedMicrobeOrganelles.Organelles;
+
+        // The hex to start the recursion with
+        var initHex = organelles[0].Position;
+
+        // List of all visited hexes
+        var checkedHexes = new List<Hex> { initHex };
+
+        // List of hexes that should be visited
+        var shouldBeChecked = organelles.Select(p => p.Position).ToList();
```

Comment:
This isn't exactly the optimal commenting style either.
Only the second comment of these would be something I'd keep. All of the other ones are / could be explained away with good variable naming.